go/teams/audit.go

```diff
@@ -394,8 +434,14 @@ func (a *Auditor) auditLocked(m libkb.MetaContext, id keybase1.TeamID, headMerkl
 	}
 
 	last := lastAudit(history)
-	if last != nil && last.MaxChainSeqno == maxChainSeqno {
-		m.CDebugf("Short-circuit audit, since there is no new data (@%v)", maxChainSeqno)
+
+	// It's possible that we're bouncing back and forth between the Fast and Slow
+	// loader. Therefore, it might have been that we previous audited up to chainlink
+	// 20, and now we're seeing an audit only for link 18 (if one of them was stale).
+	// That's fine, just make sure to short-circuit as long as we've audited past
+	// the given maxChainSeqno.
```

Comment:
Do we need to and are we asserting that the link 18 in round B is the same link as 18 was in round A?

Reply:
I think this can only happen if in step (1) we load up to 18 and are happy;  in step (2) we load up to 20 and are happy with an audit that what we saw in 1 is an extension of what we saw in 2; and in step (3) we're just reusing what we saw in (1).

I need to think a bit more that what I said about what we saw in link (2) is still true giving the short-circuiting and audit reuse that's allowed.